Participants/i-marsh.md

```diff
@@ -0,0 +1,7 @@
+### Hi, I'm i-marsh
+
+| Property | Value |
+|:---------------|:-----------------|
+| Name | Ian Marsh |
+| Favorite color | Green |
```

Comment:
I know for a fact that green _is not_ your favorite color. Could you update this please? 

![elf-throneoflies](https://user-images.githubusercontent.com/4215/38573290-981cc898-3cc3-11e8-8e26-0cca549555c3.gif)

Reply:
I have no idea what my favorite color is ... especially when it's **_colour_**